cunumeric/module.py

```diff
@@ -3232,18 +3232,68 @@ def allclose(a, b, rtol=1e-5, atol=1e-8, equal_nan=False):
     """
     if equal_nan:
         raise NotImplementedError(
-            "cuNumeric does not support equal NaN yet for allclose"
+            "cuNumeric does not support `equal_nan` yet for allclose"
         )
     args = (np.array(rtol, dtype=np.float64), np.array(atol, dtype=np.float64))
     return ndarray._perform_binary_reduction(
-        BinaryOpCode.ALLCLOSE,
+        BinaryOpCode.ISCLOSE,
         a,
         b,
         dtype=np.dtype(bool),
         extra_args=args,
     )
 
 
+@add_boilerplate("a", "b")
+def isclose(a, b, rtol=1e-5, atol=1e-8, equal_nan=False):
+    """
+
+    Returns a boolean array where two arrays are element-wise equal within a
+    tolerance.
+
+    Parameters
+    ----------
+    a, b : array_like
+        Input arrays to compare.
+    rtol : float
+        The relative tolerance parameter (see Notes).
+    atol : float
+        The absolute tolerance parameter (see Notes).
```

Comment:
There is no "Notes" section in this docstring. I think we should at least note the comparison formula used in this function, so that the documentation can stand on its own.

```
absolute(a - b) <= (atol + rtol * absolute(b))
```

Reply:
In prev PRs I copied the numpy docs verbatim where possible

https://numpy.org/doc/1.22/reference/generated/numpy.isclose.html

Reply:
522a2677c2a4ed781546c1e7023ca31e3b928c74